source/Ldap.Integration.Tests/scripts/ActiveDirectory/Azure/main.tf

```diff
@@ -0,0 +1,170 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.0"
```

Comment:
You should consider jumping up to v3 which released this week, it might break some of your file below but best to start from current, I think.